blob/src/main/java/io/crate/blob/v2/BlobIndicesService.java

```diff
@@ -130,6 +131,34 @@ public BlobShard blobShardSafe(ShardId shardId) {
         return blobShardSafe(shardId.getIndex(), shardId.id());
     }
 
+    private class LifecycleListener extends IndicesLifecycle.Listener {
+
+        @Override
+        public void afterIndexCreated(IndexService indexService) {
+            indices.put(indexService.index().getName(), new BlobIndex(globalBlobPath));
```

Comment:
There is an `@After` assertion to make sure no indices are leaked: https://github.com/crate/crate/pull/4482/files#diff-97b4abd1024416837a21d1f1d8c1a106R58